column_computation.hh

```diff
@@ -71,3 +71,23 @@ public:
     virtual bytes serialize() const override;
     virtual bytes_opt compute_value(const schema& schema, const partition_key& key, const clustering_row& row) const override;
 };
+
+
+/*
+ * Computes token value of partition key and returns it as long_type.
+ * The return type means that it can be trivially sorted (for example
+ * if computed column using this computation is a clustering key),
+ * preserving the correct order of tokens (using signed comparisons).
+ *
+ * Please use this class instead of legacy_token_column_computation.
+ * 
+ * Referred to as "version 2"/"v2" in schema.
```

Comment:
I think it's not clear what the phrase "in schema" means. Also I see below it's actually "token_v2", not "version 2" or "v2". Perhaps you can have a sentence like:
The type name stored for computations of this class is "token_v2" (the name "token" refers to the older legacy_token_column_computation).